bigchaindb/common/schema/transaction.yaml

```diff
@@ -161,30 +166,22 @@ definitions:
           uri:
             type: string
             pattern: "^cc:([1-9a-f][0-9a-f]{0,3}|0):[1-9a-f][0-9a-f]{0,15}:[a-zA-Z0-9_-]{0,86}:([1-9][0-9]{0,17}|0)$"
-      owners_after:
-        "$ref": "#/definitions/owners_list"
+      public_keys:
+        "$ref": "#/definitions/public_keys"
         description: |
-            List of public keys associated with asset ownership at the time
-            of the transaction.
-      amount:
-        type: integer
-        description: |
-            Integral amount of the asset represented by this condition.
-            In the case of a non divisible asset, this will always be 1.
-  fulfillment:
+            List of public keys associated with asset control.
```

Comment:
Maybe "List of public keys associated with the conditions on an output."

Or "List of public keys associated with the transfer conditions on an output."